app/views/posts/index.html.erb

```diff
@@ -1,11 +1,27 @@
-<p>posts is saved</p>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script type="text/javascript">
+
+    $(document).ready(function() {
+      $('#bum_toggle').change(function (){
```

Comment:
do we want a bum toggle?

Reply:
renaming it `mol_toggle` for mel and ollie